src/main/resources/application.properties

```diff
@@ -58,6 +58,7 @@ spring.mail.properties.mail.smtp.auth: false
 spring.mail.properties.mail.smtp.starttls.enable: false
 
 event.repository.url:
+event.repository.shallow:true
```

Comment:
Add a white space between the colon and "true" value, so it follows the the same format as other lines in this files.

Reply:
Sure . I will do that